_heat/Heat_Orchestration_Template_LB.yaml

```diff
@@ -1,4 +1,4 @@
-heat_template_version: 2015-10-15
+heat_template_version: 2015-10-15 (posted 7-5-2016)
```

Comment:
I don't think you want to do that, as this is a key that heat reads. The template version indicates a specific syntax to the template. See http://docs.openstack.org/developer/heat/template_guide/hot_spec.html#heat-template-version